### PR TITLE
Its 1208  - Adding google pass logic to save to mongo and add api

### PR DESCRIPTION
**Pull request by Ali-Mahmood**

### CODE REVIEW

File: app/repositories/GooglePassRepository.scala [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  domainFormat = GooglePass.mongoFormat,
+  indexes = Seq(
+    IndexModel(
+      Indexes.ascending("passId"),
+      IndexOptions().name("passId").unique(true)
+    ),
+    IndexModel(
+      Indexes.ascending("fullName", "nino"),
+      IndexOptions().name("fullName_Nino")
+    )
```
Comment (rambabu-posa) [inline app/repositories/GooglePassRepository.scala:63]: We should set time to expire here like `https://github.com/hmrc/find-my-nino-add-to-wallet/blob/main/app/repositories/ApplePassRepository.scala#L74`

File: app/services/googlepass/GooglePassUtil.scala [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  val id: String = s"hmrc-${UUID.randomUUID()}"
+  val key: String = config.googleKey
+  val expiry: Int = config.googleJWTExpiry
+
+
+  def createGooglePassWithCredentials(name: String, nino: String, googleCredentials: GoogleCredentials): String = {
+
+    val googlePassCardContent = createGooglePassCardContent(name, nino)
+    val jwt = createGenericPrivatePass.createJwtWithCredentials(id, issuerId, googlePassCardContent, googleCredentials, expiry)
+    val saveUrl = "https://pay.google.com/gp/v/save/" + jwt
```
Comment (rambabu-posa) [inline app/services/googlepass/GooglePassUtil.scala:38]: I saw some hard coded strings in this file. Is it possible to extract them into configuration or messages file?

**Review by rambabu-posa — APPROVED** [review]
LGTM

**Comment by rambabu-posa** [comment]
Please fix build failure issues and also do Squash and merge.

**Review by rishisachania — APPROVED** [review]

**Review by rambabu-posa — APPROVED** [review]

**Pull request merged**